validator/sawtooth_validator/server/component_handlers.py

```diff
@@ -178,6 +178,11 @@ def add(
         client_handlers.BlockGetByNumRequest(block_store),
         client_thread_pool)
 
+    dispatcher.add_handler(
+        validator_pb2.Message.CLIENT_BLOCK_GET_BY_TRANSACTION_REQUEST,
+        client_handlers.BlockGetByTransactionRequest(block_store),
+        thread_pool)
```

Comment:
This should use the `client_thread_pool`

Reply:
Changed, thx